package.json

```diff
@@ -21,5 +22,25 @@
   "bugs": {
     "url": "https://github.com/JorgeVQA/cypress-training/issues"
   },
-  "homepage": "https://github.com/JorgeVQA/cypress-training#readme"
+  "homepage": "https://github.com/JorgeVQA/cypress-training#readme",
+  "devDependencies": {
+    "@types/jest": "^27.4.1",
```

Comment:
why did you need the @types/jest dependency?

Reply:
When changed extension to test file from .js to .ts some reserved words were not recognized in the code, so I took suggestions where errors marked the code and there was a command I run and that was why that dependency was added.   I will check on Monday with you, Thanks.

Reply:
Like improve, you would set up these dependencies in your project, in this way you ensure that in other machine is set up properly

![Improve_packageFile](https://user-images.githubusercontent.com/22506260/164873995-36e01e68-6ef5-416d-9b90-6ed4f86e574c.PNG)

-- try to run --
![Not_Run_project_In_Other_Machine](https://user-images.githubusercontent.com/22506260/164873994-372c2e5b-ac9a-4a87-b722-2807bf551a96.PNG)

>>> 3 Doritos after >>>
![Dependencies_Save](https://user-images.githubusercontent.com/22506260/164874226-a7cadf88-566c-42aa-a37d-38c4d69453c2.PNG)

![Run_Test](https://user-images.githubusercontent.com/22506260/164874230-93cc7082-9b83-494e-83eb-8ba4ee7a7f9d.PNG)

Reply:
Fixed, I will update PR with a new commit.